testRenamePackage.sh

```diff
@@ -4,6 +4,6 @@ cd ..
 rm -rf android-starter-project-copy
 cp -R android-starter-project android-starter-project-copy
 cd android-starter-project-copy
-./renamePackage.sh com.demo.mobile
+python renamePackage.py com.demo.mobile
```

Comment:
eh, technically yes it would work, but Windows doesn't use the `./` part. Also Unix & Dos file execution permissions are different, which means if a future PR of `renamePackage.py` is made from a windows machine, it could erase the unix style file permissions.

Using `python renamePackage.py` is perfectly universal between Windows and Mac